### DISCUSSION TITLE
Why shadow-[0px_1px_16px_0px_rgba(0,0,0,0.16)] parse to shadow-[0px_1px_16px_0px_rgba(02c 0\2c 0\2c 0.16)]

### DISCUSSION THREAD

**Discussion by ghost** [tailwindlabs/tailwindcss#14681] [Help]
shadow-[0px_1px_16px_0px_rgba(0,0,0,0.16)] parse to shadow-[0px_1px_16px_0px_rgba(02c 0\2c 0\2c 0.16)]
 The first comma in rgba() should be parse as \2c, but \ is lost, so this class is not work.
WHY
![image](https://github.com/user-attachments/assets/e17290e4-5d17-4189-af85-b44a04e2f6eb)

![image](https://github.com/user-attachments/assets/ac7ac774-140b-4591-8aa6-6edb49a68a8e)

**Comment by adamwathan** [comment]
Can you share a reproduction?

**Reply by ghost** [reply]
https://github.com/AriesLyt/public-help-question/tree/main/tailwindcss-rgb-help

I reproduced this error
class bg-[rgb(0,0,0)] is not working
